utilities/ingest/dataflow_hl7v2_to_hl7v2/README.md

```diff
@@ -0,0 +1,52 @@
+# hl7V2Store to hl7V2Store Data Pipeline
+
+> [!NOTE]
+> This is non-production demo/example code. It comes with no
+guarantees and requires modification before use in any stable, durable or SLA'ed environment.
+
+## Overview
+
+The purpose of the hl7v2-to-hl7v2 pipeline is to stream HL7 V2 messages
+from one hl7V2Store to another using a source hl7V2Store, its attached Pub/Sub Subscription and a destination hl7V2Store.
+
+![image](images/dataflow-hl7v2-to-hl7v2-architecture.png)
+
+## Environment
+This pipeline is agnostic to the location where the source and destination
+hl7V2Stores sit. To read from source, it relies on the name of the hl7V2Store provided in the `name`
+element of each Pub/Sub message.
+
+Before running this pipeline, ensure the Dataflow service account has the following IAM
+bindings in addition to normal Dataflow IAM:
+- roles/healthcare.hl7V2StoreViewer on the source Store
```

Comment:
Link these to GCP Public docs of the roles?

Reply:
Linked